clang/test/Driver/hexagon-default-build-attributes.s

```diff
@@ -0,0 +1,20 @@
+/// Enabled by default for assembly
+// RUN: %clang -target hexagon-unknown-elf -### %s 2>&1 \
```

Comment:
`--target=` for new tests.
`-target ` has been deprecated since Clang 3.4

Reply:
Done